pipelines/respondent-home-ui.yml

```diff
@@ -211,8 +211,8 @@ jobs:
       environment_variables:
         HOST: 0.0.0.0
         LOG_LEVEL: INFO
-        ACCOUNT_SERVICE_URL: https://ohs-alpha.onsdigital.uk
-        EQ_URL: https://eq.ons.gov.uk/session?token=
+        ACCOUNT_SERVICE_URL: # respondent home domain TBC
```

Comment:
Approving on the assumption that this will is ok as just a comment and will be addressed soon.

Reply:
👍 Will be set as soon as we know the domain